ldms/src/sampler/netlink/input/test_slps.sh

```diff
@@ -30,6 +30,9 @@ set_kokkos () {
 	export KOKKOS_LDMS_RECONNECT=59
 }
 
+timeout=1m
+export ZAP_MAX_IO=8
```

Comment:
@baallan, there is no ZAP_MAX_IO environment variable.

Reply:
Did you mean to say "use ZAP_IO_MAX instead", which appears to be influential but undocumented?